### ISSUE TITLE
Saving something in settings panel redirects to "install" url

### ISSUE THREAD

**Issue by astronauta88** [craftcms/cms#3129]
### Description
Sometime, saving something in settings page, for examle section field layout, redirects me toward this url, where "craf url" is name of folder craft is installed in:

`http://localhost/craft_url/web/admin/install`

This url produces error:

`Template not found: install`

When i go back to other url, everything seems to be ok.

### Additional info

- Craft version: Craft CMS 3.0.16.1
- PHP version: 7.2.2
- Plugins & versions: 
Command Palette	3.1.4
Position Fieldtype	1.0.13
Redactor	2.1.2

**Comment by astronauta88** [comment]
@brandonkelly
No, not really. But it seems to me that it hapens when i quickly try to open another CP site after saving something in settings. When i wait few seconds, it doesnt seems to happen.

**Comment by brandonkelly** [comment]
The only time Craft would ever redirect to `/admin/install` on its own accord is if it thinks it’s not installed yet, and if Dev Mode is enabled.

Can you try disabling Dev Mode and see if this continues to happen? You should start getting 503 errors instead, if Craft actually thinks it’s not installed from time to time.
